Prog/get-zillow.R

```diff
@@ -0,0 +1,125 @@
+
+library(tidyverse)
+library(lubridate)
+
+# Inventory -------------------------------------------------------
+
+
+inventory <- read_csv('http://files.zillowstatic.com/research/public/County/MonthlyListings_SSA_AllHomes_County.csv')
+inventoryMSA <- read_csv('http://files.zillowstatic.com/research/public/Metro/MonthlyListings_SSA_AllHomes_Metro.csv')
+
+#county %in% c('District of Columbia','Charles','Frederick','Montgomery','Prince Georges','Arlington', 'Fairfax', 'Fairfax City', 'Loudoun','Prince William', 'Alexandria City', 'Falls Church City', 'Manassas City', 'Manassas Park City')
+
+inventoryCOGmonth <- inventory %>% 
+  select(-RegionType, -SizeRank, -RegionID) %>% 
+  rename(county = RegionName, state = StateName) %>% 
+  gather(key = 'monthx', value = 'inventoryCOG', -state, -county) %>% 
+  filter(state %in% c('DC', 'MD', 'VA'),
+         county %in% c('District of Columbia','Charles County','Frederick County','Montgomery County','Prince Georges County','Arlington County', 'Fairfax County', 'Fairfax City', 'Loudoun County','Prince William County', 'Alexandria City', 'Falls Church City', 'Manassas City', 'Manassas Park City')
+         ) %>%
+  arrange(county, monthx) %>% 
+  mutate(month = substr(monthx, 6, 7),
+         year= substr(monthx, 1,4))
+
+inventoryMSAJuly <- inventoryMSA %>% 
+  select(-RegionType, -SizeRank, -RegionID, -StateName) %>% 
+  rename(Metro = RegionName) %>% 
+  gather(key = 'monthx', value = 'inventoryMetro', -Metro) %>% 
+  filter(Metro=="Washington, DC") %>% 
+  arrange(Metro, monthx) %>% 
+  mutate(month = substr(monthx, 6, 7),
+         year= substr(monthx, 1,4)) %>% 
+  filter(month=="07") %>% 
+  select(-monthx, -Metro, -month)
+
+#### inventory for COG region
+inventoryCOGJuly <- inventoryCOGmonth %>% 
+    filter(month=="07") %>% 
+    group_by(year) %>% 
+    summarize(JulyInventory= sum(inventory))
+
+
+# Sale prices --------------------------------------------------------
+
+price <- read_csv('http://files.zillowstatic.com/research/public/Metro/Sale_Prices_Msa.csv')
+
+priceMSAJuly <- price %>% 
+  select(-RegionID, -SizeRank) %>% 
+  rename(name = RegionName) %>% 
+  gather(key = 'monthx', value = 'Mediansaleprice', -name) %>% 
+  filter(name =="Washington, DC") %>%
+  mutate(month = substr(monthx, 6, 7), 
+         year= substr(monthx, 1,4)) %>% 
+  filter(month=="07") %>% 
+  select(-name, -monthx, -month)%>% 
+  arrange(year, Mediansaleprice, -month)
```

Comment:
this doesn't run either, i think you need to delete `, -month` from the `arrange()` statement